### PR TITLE
Make minor cleanup and improve readability

### PR DESCRIPTION
**Pull request by imanhodjaev**
This small PR includes the following changes

* Use set literal instead of initiating set,
* Make `__str__` in `compat.py` reachable,
* Remove class definitions with empty parens.

### CODE REVIEW

File: apistar/compat.py [hunk 1]
```diff
@@ -38,8 +38,8 @@ def __repr__(self):
             mode = "" if self.closed else " '%s'" % self.file.mode
             return "<DownloadedFile '%s', %s%s>" % (self.name, state, mode)
 
-            def __str__(self):
-                return self.__repr__()
+        def __str__(self):
+            return self.__repr__()
```
Comment (Bogdanp) [inline apistar/compat.py]: Why not `repr(self)`?

Comment (imanhodjaev) [reply apistar/compat.py]: I just wanted to keep it as it is, so in next PRs I or someone else can do it.

**Comment by tomchristie** [comment]
Great, yup!

**Comment by tomchristie** [comment]
One thing we could consider for formatting aspects would be using ‘black’ to enforce a common style throughout. I don’t know how that looks in practice, or if it’s sonething we definitely want, but might be worth is taking a look at.

**Comment by imanhodjaev** [comment]
Great then I will try to configure `black` and add formatting script for it.

**Pull request merged**